packages/js/src/initializers/block-editor-integration.js

```diff
@@ -114,6 +114,8 @@ function registerFills( store ) {
 	const blockSidebarContext = { locationContext: "block-sidebar" };
 	const blockMetaboxContext = { locationContext: "block-metabox" };
 
+	const showJetpackBoostAd = get( window, "wpseoScriptData.canShowJetpackBoostAd", false );
```

Comment:
Why didn't you put this inside the `JetpackBoost`, along side the other window retrievals and return `null` check?